src/include/sof/audio/module_adapter/module/generic.h

```diff
@@ -1,11 +1,12 @@
-/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Lets find a better name for "generic" - it says nothing at all
module_base.h ?

Reply:
yep, agree. base.h or core.h are more meaningfull

Reply:
renamed to base.h